src/dotnet/commands/dotnet-test/LocalizableStrings.cs

```diff
@@ -32,7 +32,7 @@ internal class LocalizableStrings
         public const string CmdLoggerOption = "LoggerUri/FriendlyName";
 
         public const string CmdLoggerDescription = @"Specify a logger for test results. 
-                                        Example: --logger:trx";
+                                        Example: --logger ""trx[;LogFileName=<Defaults to unique file name>]""";
```

Comment:
I'd prefer this be a separate PR. If we get another loc chance then we'll merge, otherwise it's a nice-to-have.